src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
block should have class `stars` instead `star`